src/bp/core/misc/code-sandbox.ts

```diff
@@ -91,3 +93,18 @@ export class SafeCodeSandbox {
     this.tmpDir.removeCallback()
   }
 }
+
+export class UntrustedSandbox {
+  @memoize(() => getCacheKeyInMinutes(5)) // cache the result of this function for 1 minute
+  static getSandboxProcessArgs() {
+    const exposedEnv = {
+      ..._.pickBy(process.env, (_value, name) => name.startsWith('EXPOSED_')),
+      ..._.pick(process.env, 'TZ', 'LANG', 'LC_ALL', 'LC_CTYPE')
```

Comment:
Should we document them in BotpressEnvironmentVariable? i don't know what these LC_ vars are, since we will soon add the typing in the code editor, that would be nice !

Reply:
These are mostly specific to unix-based systems I think so not sure if we should indeed document them